src/card_operations/card_fetch_wallet_list.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file    card_fetch_wallet_list.h
+ * @author  Cypherock X1 Team
+ * @brief   Header file exporting function to fetch wallet list from X1 card
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ */
+#ifndef CARD_FETCH_WALLET_LIST_H
+#define CARD_FETCH_WALLET_LIST_H
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+#include <stdbool.h>
+#include <stdint.h>
+
+/*****************************************************************************
+ * MACROS AND DEFINES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * TYPEDEFS
+ *****************************************************************************/
+typedef struct {
+  uint8_t acceptable_cards;
+  const char *heading;
+  const char *msg;
+} card_fetch_wallet_list_config_t;
```

Comment:
see if we can add `skip_card_removal` useful for the last card taps in the flow

Reply:
Added, thanks